client/extensions/woocommerce/app/products/product-form-details-card.js

```diff
@@ -61,18 +61,20 @@ export default class ProductFormDetailsCard extends Component {
 						/>
 					</FormLabel>
 				</div>
-				<div>
+				<div className="products__product-form-details-wrapper">
+					<div className="products__product-form-details-images">
 
-				</div>
-				<div>
-					<FormLabel>
-						<span>{ __( 'Product name' ) }</span>
-						<FormTextInput value={ product.name || '' } onChange={ this.setName } />
-					</FormLabel>
-					<FormLabel>
-						<span>{ __( 'Description' ) }</span>
```

Comment:
Shouldn't we use fieldsets here? Instead of span, this should be a label. See 'Form Text Input' examples here: https://wpcalypso.wordpress.com/devdocs/design/form-fields or any of the forms in /settings.

This will also fix some of the tight spacing here.